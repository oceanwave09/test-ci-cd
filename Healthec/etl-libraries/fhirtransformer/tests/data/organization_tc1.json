{
  "sub_group_id": "NECQ1032",
  "tax_id": "983748232",
  "active": "true",
  "name": "Good Health Clinic",
  "type_code": "prov",
  "type_display": "Healthcare Provider",
  "street_address_1": "132, Erewhon St",
  "city": "Kingston",
  "state": "New York",
  "zip": "12401",
  "phone_mobile": "(03) 3410 5613",
  "email": "michael.jackson@gmail.com",
  "parent_organization_id": "bb7a0594-6be2-44e9-a1d4-dab0c74f91aa"
}
