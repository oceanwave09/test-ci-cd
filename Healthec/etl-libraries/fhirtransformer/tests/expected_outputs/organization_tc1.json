{
  "resourceType": "Organization",
  "identifier": [
    {
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "TAX",
            "display": "Tax ID number"
          }
        ]
      },
      "system": "urn:oid:2.16.840.1.113883.4.4",
      "value": "983748232"
    },
    {
      "type": {
        "coding": [
          {
            "system": "http://terminology.hl7.org/CodeSystem/v2-0203",
            "code": "RI",
            "display": "Resource Identifier"
          }
        ]
      },
      "system": "http://healthec.com/identifier/practice/sub_group_id",
      "value": "NECQ1032"
    }
  ],
  "active": true,
  "type": [
    {
      "coding": [
        {
          "system": "http: //terminology.hl7.org/CodeSystem/organization-type",
          "code": "prov",
          "display": "Healthcare Provider"
        }
      ]
    }
  ],
  "name": "Good Health Clinic",
  "partOf": {
    "reference": "Organization/bb7a0594-6be2-44e9-a1d4-dab0c74f91aa"
  },
  "telecom": [
    {
      "system": "phone",
      "use": "mobile",
      "value": "(03) 3410 5613"
    },
    {
      "system": "email",
      "value": "michael.jackson@gmail.com"
    }
  ],
  "address": [
    {
      "line": [
        "132, Erewhon St"
      ],
      "city": "Kingston",
      "state": "New York",
      "postalCode": "12401"
    }
  ]
}